projects/core/src/cart/store/reducers/wish-list.reducer.ts

```diff
@@ -0,0 +1,15 @@
+import { CartActions } from '../actions';
+
+export const initialState = '';
+
+export function wishListReducer(
```

Comment:
How about move it into `multi-cart.reducer`?

Reply:
When language/currency changed, does multi-cart data need to be reloaded?

Reply:
I did think about moving it there will do. It should be reloaded I will make sure it's happening.

Reply:
Moved, site context change will be handled in #5231